src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -25,12 +25,24 @@ private enum ParameterSyntaxKind : byte
             DefaultParameter = 4,
         }
 
+        private class LazyDefaultValue
```

Comment:
Consider a more descriptive name here.  LazyDefaultValue can mean a lot of things, doesn't seem to capture the intent here.